add contributing guidelines

BigChief45 opened the pull request.

jd commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,28 @@
+
+### Issues
+
+We use the [Gihub issue tracker](https://github.com/gnocchixyz/gnocchi/issues) for reporting issues. Before opening a new issue, ensure the bug was not already reported by searching on Issue tracker first.
```
issue without maj

jd commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,28 @@
+
+### Contributing to Documentation
+
+When contributing to the documentation, please add `[ci skip]` to your documentation commit message to skip CI build on that commit. Please remember to use it for commits containing only documentation changes.
```
We actually don't want to skip the CI on doc update. We test various things since the web site is built from that.

jd commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,28 @@
+
+Tests are run using [tox](https://tox.readthedocs.io/en/latest/). Tox creates a virtual environment for each test environment, so make sure you are using an up to date version of [virtualenv](https://pypi.python.org/pypi/virtualenv).
+
+Different test environments and configurations can be found in [`tox.ini`](https://github.com/gnocchixyz/gnocchi/blob/master/tox.ini). For example, to run tests with Python 2.7, PostgreSQL as indexer, and file as storage backend:
```
`tox -l` provides the list of env available

BigChief45 commented:
I have squashed the commits and added the new RST file to `index.rst`.

Also, I think you can squash merge pull requests too:

![image](https://cloud.githubusercontent.com/assets/6859247/26384721/b52885ec-406d-11e7-8f43-7a6674d083b5.png)
